run_scripts/run_pipeline.sh

```diff
@@ -28,6 +28,6 @@ DATA_BACKUPS_DIR=$8
 
 ./4_generate_outputs.sh "$USER" "$GOOGLE_CLOUD_CREDENTIALS_FILE_PATH" "$PIPELINE_CONFIGURATION" "$DATA_ROOT"
 
-./5_coda_add.sh "$CODA_PUSH_CREDENTIALS_PATH" "$CODA_TOOLS_ROOT" "$DATA_ROOT"#
+./5_coda_add.sh "$CODA_PUSH_CREDENTIALS_PATH" "$CODA_TOOLS_ROOT" "$DATA_ROOT"
```

Comment:
Drive by comment - the renumbering here means they're now different across pipelines. I had handled this in the past by renaming 2_fetch_raw_data.sh -> 2_fetch_data.sh, and having that stage run the normal and recovery data fetches. Not sure which is best.

Reply:
@as2388 merging into 2_fetch_data.sh is a neater approach. Updated thanks. PTAL!